### PR TITLE
Add day of the week to room booking details

### PR DESCRIPTION
**Pull request by GovernmentPlates**
Closes: #5718 

Previews:
image
image

### CODE REVIEW

File: indico/modules/rb/client/styles/palette.scss [hunk 1]
```diff
@@ -47,6 +47,7 @@ $pending-cancellation-fadeout-colors: $pending-cancellation-color,
   lighten($pending-cancellation-color, 30%);
 $rejection-fadeout-colors: $rejection-color, lighten($rejection-color, 30%);
 $other-booking-fadeout-colors: $other-booking-color, lighten($other-booking-color, 30%);
+$booking-details-dates-weekday-color: #8f8f8f;
```
[the remark this answers is not in the document]
Comment (GovernmentPlates) [reply indico/modules/rb/client/styles/palette.scss:50]: I found the default `$gray` in the pallet makes the text slightly harder to read, which is why I used a darker shade of gray for this (I've attached what it looks like when using the default style).

image

[the remark this answers is not in the document]
Comment (ThiefMaster) [reply indico/modules/rb/client/styles/palette.scss:50]: What about `$dark-gray` instead?

File: indico/modules/rb/client/js/components/TimeInformation.jsx [hunk 2]
```diff
@@ -49,11 +49,25 @@ function TimeInformation({
...
+                  name="endDateDay"
+                  wrapper={<span styleName="booking-times-weekday" />}
+                  value={mEndDate.format('ddd.')}
+                />{' '}
                 <Param name="endDate" wrapper={<strong />} value={mEndDate.format('L')} />
               </Translate>
             ) : (
-              <strong>{mStartDate.format('L')}</strong>
+              <strong>
+                <span styleName="booking-times-weekday">{mStartDate.format('ddd.')}</span>{' '}
```
Comment (ThiefMaster) [inline indico/modules/rb/client/js/components/TimeInformation.jsx:68]:
```suggestion
                <span styleName="booking-times-weekday">{mStartDate.format('ddd')}</span>{' '}
```

File: indico/modules/rb/client/js/components/TimeInformation.jsx [hunk 3]
```diff
@@ -49,11 +49,25 @@ function TimeInformation({
             <Icon name="calendar outline" />
             {mEndDate && !mStartDate.isSame(mEndDate, 'day') ? (
               <Translate>
-                <Param name="startDate" wrapper={<strong />} value={mStartDate.format('L')} /> to{' '}
+                <Param
+                  name="startDateDay"
+                  wrapper={<span styleName="booking-times-weekday" />}
+                  value={mStartDate.format('ddd.')}
```
Comment (ThiefMaster) [inline indico/modules/rb/client/js/components/TimeInformation.jsx:55]:
```suggestion
                  value={mStartDate.format('ddd')}
```

this is included in `ddd` for languages that use it. currently you end up with this in french:

image

File: indico/modules/rb/client/js/components/TimeInformation.jsx [hunk 4]
```diff
@@ -49,11 +49,25 @@ function TimeInformation({
...
+                />{' '}
+                <Param name="startDate" wrapper={<strong />} value={mStartDate.format('L')} />
+                {' — '}
+                <Param
+                  name="endDateDay"
+                  wrapper={<span styleName="booking-times-weekday" />}
+                  value={mEndDate.format('ddd.')}
```
Comment (ThiefMaster) [inline indico/modules/rb/client/js/components/TimeInformation.jsx:62]:
```suggestion
                  value={mEndDate.format('ddd')}
```

File: indico/modules/rb/client/js/common/bookings/BookingDetails.jsx [hunk 5]
```diff
@@ -292,7 +292,7 @@ class BookingDetails extends React.Component {
         rejections: rejections[day] || [],
         other: other[day] || [],
       },
-      label: moment(day).format('L'),
+      label: moment(day).format('ddd. L'),
```
Comment (ThiefMaster) [inline indico/modules/rb/client/js/common/bookings/BookingDetails.jsx:295]:
```suggestion
      label: moment(day).format('ddd L'),
```

**Pull request merged**